sky/backends/wheel_utils.py

```diff
@@ -143,9 +143,13 @@ def _build_sky_wheel() -> pathlib.Path:
                            stdout=subprocess.DEVNULL,
                            stderr=subprocess.PIPE,
                            check=True)
-        except subprocess.CalledProcessError as e:
-            raise RuntimeError('Failed to build pip wheel for SkyPilot. '
-                               f'Error message: {e.stderr.decode()}') from e
+        except (subprocess.CalledProcessError, FileNotFoundError) as e:
+            if isinstance(e, FileNotFoundError):
+                msg = 'pip3 not found. Please install pip3.'
```

Comment:
Hmm, this may not actually be a pip issue? Would it be possible to identify the issue in a better way?

Also, for the message itself, we should give some instructions for how to resolve it, e.g., linking to a FAG docs page in SkyPilot docs for how to resolve such issue.

Reply:
Yeah, this is about pip not being installed at all in some environments (e.g., `uv venv` without `--seed`
creates a pure UV environment without pip).

The latest commits already address this by:
1. Using `python -m pip` instead of `pip3`
2. Detecting if it's UV/conda and suggesting specific commands (`uv pip install pip` or `conda install pip`)

Happy to add a docs link if you think that would be helpful, but the error messages now provide the exact commands users need.